src/ralph/admin/widgets.py

```diff
@@ -156,14 +161,63 @@ def get_related_url(self, info, action, *args):
 
     def value_from_datadict(self, data, files, name):
         value = data.get(name)
-        if getattr(self, 'multi', False) is True:
-            print('name', name)
+        if self.multi:
             if value:
-                value = value.split(',')
+                value = value.split(self.separator)
             else:
                 value = []
         return value
 
+    def get_search_fields(self):
+        searched_fields_tooltip = defaultdict(list)
+        polymorphic_descendants = getattr(
+            self.rel_to, '_polymorphic_descendants', []
+        )
+
+        if polymorphic_descendants:
+            # Check if model after which we are looking for is polymorphic
+            # if they are also looking for the models of its dependencies
+            # or by limit_models defined in model
+            polymorphic_models = polymorphic_descendants
+            limit_models = getattr(self.field, 'limit_models', [])
```

Comment:
could you add some comment, that usually all polymorphic descendants are processed here, but they could be limited only to particular models using `limit_models` attribute of field?